scst-store/amr/data-model-and-concepts.hbs.md

```diff
@@ -10,28 +10,58 @@ The following diagram shows the data models used in the AMR to store artifact in
+- `Containers`
+- `Images`
+- `Commits`
+- `AppAcceleratorRuns`
+- `AppAcceleratorFragments`
+- `DoraMetrics`
+- `DoraMetricsPerCorrelationID`
 
 ### <a id='locations'></a> Locations
 
 The `Locations` data model stores data about the locations that `Apps` are deployed on, like clusters.
 
 - `reference`: unique reference to the location. It is automatically set to be the `kube-system` namespace UID by the AMR. This is not configurable by the user.
-- `alias`: unique alias for the location. 
-  By default, it is initialized with the value of the `reference`, but users have the option to configure it to a different value. See [AMR Configuration](configuration.hbs.md).
-- `labels`: labels of the location. Users can add them by using tap values YAML file. See [AMR Configuration](configuration.hbs.md).
\ No newline at end of file
+- `labels`: labels of the location. Users can add them by using tap values YAML file. See [AMR Configuration](configuration.hbs.md).
+
+### <a id='containers'></a> Containers
+
+The `Containers` data model represents the information of a container which is a runtime information about an image. The Observer sends this information to the Cloud Event Handler, so each entry represents the state of the container when an event occurred. See [cloud events](cloudevents.hbs.md).
+These entries correspond to events, such as when a container is running or terminated.
+This lets users see the history of their containers.
+
+Each `Container` data entry stores information about the associated location in which it was deployed, details about the state, and what image was used. See the diagram above for a list of all fields the data model stores.
+You can only associate an `Containers` entry with one `Locations` entry. You can point multiple `Containers` entries to the same `Locations` entry. 
+
+### <a id='images'></a> Images
+
+The `Images` data model represents the information of an image which acts as a template for containers. The Observer sends this information to the Cloud Event Handler to store newly generate image. See [cloud events](cloudevents.hbs.md).
+
+Each `Image` data entry stores information about the associated location in which it was deployed, details about the state, and what image was used. See the diagram above for a list of all fields the data model stores.
+You can only associate an `Images` entry with one `Locations` entry. You can point multiple `Images` entries to the same `Locations` entry. 
+
+### <a id='commits'></a> Commits
+
+The `Commits` data model represents the information of a snapshot of the changes that have been made to a project's files. The Observer sends this information to the Cloud Event Handler to store new commits. See [cloud events](cloudevents.hbs.md).
+
+Each `Commit` data entry stores information about the associated location in which it was deployed, details about the state, and what image was used. See the diagram above for a list of all fields the data model stores.
```

Comment:
Just doublechecking: `what image was used` - is this description accurate for commits?

Reply:
I think that was something suggested by @ohlookadollar

Reply:
What is the relationship between the commit and "the image used"? I'm just not clear on what that means.

Reply:
Ah, I can make it more explicit `image` here is commit sha or tag.